rusqlite_migration/README.md

```diff
@@ -108,6 +108,12 @@ fn migrations_test() {
 
 Contributions (documentation or code improvements in particular) are welcome, see [contributing](https://cj.rs/docs/contribute/)!
 
+### Code Coverage
+
```

Comment:
This is not needed any more, I think?

Reply:
Yes, folks will find the executed command in the yaml file or in the github action run details if needed.